test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+
+    public class Startup
```

Comment:
Seems odd that this is here but not used by anything in this file. I think I'd prefer it be with `ServerFixture` if it is intended to represent the app that `ServerFixture` uses.